website/docs/guides/migration/versions/04-upgrading-to-dbt-utils-v1.0.md

```diff
@@ -68,17 +68,17 @@ models:
     tests:
       - dbt_utils.expression_is_true:
           expression: "col_a + col_b = total"
-					#replace this...
+          #replace this...
           condition: "created_at > '2018-12-31'" 
 
   - name: new_syntax
     tests:
       - dbt_utils.expression_is_true:
           expression: "col_a + col_b = total"
-					# ...with this...
+          # ...with this...
           where: "created_at > '2018-12-31'"
 ```
-
+Note that this may cause some tests to get the same autogenerated names. To resolve this, you can [define a custom name for a test](https://docs.getdbt.com/reference/resource-properties/tests#define-a-custom-name-for-one-test).
```

Comment:
```suggestion
Note that this may cause some tests to get the same autogenerated names. To resolve this, you can [define a custom name for a test](/reference/resource-properties/tests#define-a-custom-name-for-one-test).
```